live-build/misc/ansible-roles/appliance-build.qa-internal/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
What do you think about adding a comment that describes that this file is for QA specific testing items and frameworks and should not include anything that would be required by customers?

Reply:
Sounds good to me, will do